playbulb-candle/app.js

```diff
@@ -3,11 +3,13 @@ var r = g = b = 255; // White by default.
 document.querySelector('#connect').addEventListener('click', function() {
   document.querySelector('#state').classList.add('connecting');
   playbulbCandle.connect()
-  .then(() => {
+  .then(_ => {
```

Comment:
Out of curiosity, why use `_` instead of `()`? Just to be clearer that there is an argument we're ignoring?